src/nn/context.ts

```diff
@@ -60,6 +61,9 @@ export class MLContext {
   /** @internal */
   // Expose tf.js for backend debugging.
   get tf(): unknown {
+    // Set directory of wasm binaries for 'wasm' backend
+    wasm.setWasmPaths(
+        'https://webmachinelearning.github.io/webnn-polyfill/dist/');
```

Comment:
BTW, do you think we can set the path to TensorFlow.js URL
```
https://unpkg.com/@tensorflow/tfjs-backend-wasm@${tf.version_core}/dist/
```

So we can avoid host our own version.

Reply:
Good idea! I wasn't aware that there're official wasm binaries could be referenced. I will update it soon.